packages/trpc/server/routers/viewer/availability/team/listTeamAvailability.handler.ts

```diff
@@ -120,9 +122,16 @@ async function getInfoForAllTeams({ ctx, input }: GetOptions) {
     limit,
   });
 
+  // Get total team count across all teams the user is in (for pagination)
+
+  const totalTeamMembers =
+    await prisma.$queryRaw<number>`SELECT COUNT(DISTINCT "userId")::integer from "Membership" WHERE "teamId" IN (${Prisma.join(
+      teamIds
+    )})`;
```

Comment:
You cant count distinct when using IN in prisma client for some reason. Had to go with the raw approach